spec/models/ticket_spec.rb

```diff
@@ -88,11 +88,11 @@
              user: user,
              ticket: ticket,
              quantity: 20)
-      expect(ticket.quantity_bought_by(user)).to eq(20)
+      expect(ticket.quantity_bought_by(user, 'f')).to eq(20)
```

Comment:
you still need to fix the `'f'`s here in the specs :)

Question: i don't see any tests for when `unpaid` is `true` instead of false...